lib/spack/spack/test/compilers/basics.py.orig

```diff
@@ -0,0 +1,1028 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
```

Comment:
@climbfuji it looks like a couple extra files made it into the PR

Reply:
oops, should be fixed now. sorry for the glitch